controllers/breach-details.js

```diff
@@ -2,9 +2,13 @@
 
 const HIBP = require("../hibp");
 const DB = require("../db/DB");
+const AppConstants = require("../app-constants");
 const { changePWLinks } = require("../lib/changePWLinks");
 const { getAllEmailsAndBreaches } = require("./user");
 
+const EXPERIMENTS_ENABLED = (AppConstants.EXPERIMENT_ACTIVE === "1");
+const { getExperimentFlags } = require("./utils");
```

Comment:
Adding experiment flags to the breach details template to show/hide the Share CTA in page.